geopandas/plotting.py

```diff
@@ -336,7 +345,16 @@ def plot_series(s, cmap=None, color=None, ax=None, figsize=None, **style_kwds):
 
     if ax is None:
         fig, ax = plt.subplots(figsize=figsize)
-    ax.set_aspect("equal")
+
+    if set_aspect == "auto":
+        if s.crs and s.crs.is_geographic:
+            bounds = s.total_bounds
+            y_coord = np.mean([bounds[1], bounds[3]])
+            ax.set_aspect(1 / np.cos(y_coord * np.pi / 180))
```

Comment:
do we have some reference for this formula for future readers?

Reply:
It is taken from `sf` and I haven't found a source of it (looking for it now).